scripts/test-runner.js

```diff
@@ -247,5 +253,79 @@ function run() {
   return failed === 0;
 }
 
-if (!run())
-  process.exit(1);
+
+// Object to store all command line arguments
+class ProgramArgs {
```

Comment:
Since there shouldn't be methods on this class, you could also specify this as a flow struct:

type ProgramArgs = {|
  verbose?: boolean,
  filter?: string,
|}; then just pass the minimist return value through (or maybe still create an object literal if Flow can't figure out types.